pkg/operator/discover/discover.go

```diff
@@ -77,6 +79,14 @@ func (d *Discover) Start(namespace, discoverImage, securityAccount string) error
 
 func (d *Discover) createDiscoverDaemonSet(namespace, discoverImage, securityAccount string) error {
 	privileged := true
+	discovery_parameters := []string{"discover",
+		"--discover-interval", getEnvVar(discoverIntervalEnv, "60m")}
+	use_cv, err := strconv.ParseBool(getEnvVar(discoverUsesCV, "false"))
+	if err == nil {
```

Comment:
If you only tread `err == nil` how do you handle errors?

Reply:
Changed because the @travisn  suggections. `use-ceph-volume` is not an env-var anymore.